packages/ui-kit/src/components/rss/rss-feeds.tsx

```diff
@@ -46,41 +58,51 @@ async function fetcher(...args) {
   return Promise.all(promises);
 }
 
-export const transformData = (data) => {
+type DatedData = { pubDate: string };
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const transformData = (data: any) => {
   // First, we need to get the oldest release note from each feed,
   // which is always the last entry of the list.
   const lastEntryOfList = data
-    .reduce((list, feed) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    .reduce((list: any, feed: string | any[]) => {
```

Comment:
FYI: usually you pass a type argument to `reduce`, instead of typing each function argument.

Reply:
thanks. Properly typed the whole RSS thing, inference led to not needing a lot of signature on the reducers at all